tools/topology/m4/muxdemux.m4

```diff
@@ -2,6 +2,24 @@ divert(-1)
 
 dnl Define macro for demux widget
 
+define(comp_type, 0)
+define(sof_magic, 0x00464F53)
+define(stream_data_size, 16)
```

Comment:
are we limited to 16 bits audio data?

Reply:
oh this/these are the struct sizes needed for the blob total size, maybe better naming should be used..